src/Raven.Server/Documents/ETL/EtlProcess.cs

```diff
@@ -1013,7 +1075,7 @@ public override OngoingTaskConnectionStatus GetConnectionStatus()
                             var ravenEtlItem = testScript.IsDelete ? new RavenEtlItem(tombstone, docCollection, EtlItemType.Document) : new RavenEtlItem(document, docCollection);
 
                             var results = ravenEtl.Transform(new[] {ravenEtlItem}, context, new EtlStatsScope(new EtlRunStats()),
-                                new EtlProcessState());
+                                new EtlProcessState{SkippedTimeSeriesDocs = new HashSet<string> {testScript.DocumentId}});
```

Comment:
So it will process the time-series as well

Reply:
But it it mandatory to set it? If we have `addTimeSeries()` call it should add a time series anyway, right?

Reply:
If the user define `loadBehaviorloadTimeSeriesOf<CollectionName>Behavior(docId, timeSeriesName)` we will not run it in document change because the decision was to process time-series only on time-series change unless there is skipped doc. So the testScript will not test it.

Reply:
Okay, I see. Makes sense